Cargo.toml

```diff
@@ -13,18 +13,18 @@ categories = ["database", "api-bindings"]
 keywords = ["qdrant", "vector-search", "search-engine", "client", "grpc"]
 
 [dependencies]
-tonic = { version = "0.9.2", features = ["tls", "tls-roots"] }
-prost = "0.11.9"
-prost-types = "0.11.9"
+tonic = { version = "0.11.0", features = ["tls", "tls-roots"] }
+prost = "0.12.4"
+prost-types = "0.12.4"
 anyhow = "1"
 
 serde = { version = "1", features = ["derive"], optional = true }
 serde_json = { version = "1", optional = true }
-reqwest = { version = "0.11.24", optional = true, default-features = false, features = ["stream", "rustls-tls"] }
+reqwest = { version = "0.12.3", optional = true, default-features = false, features = ["stream", "rustls-tls"] }
```

Comment:
`reqwest` [0.12.0](https://github.com/seanmonstar/reqwest/releases/tag/v0.12.0) has extracted the http2 module in a dedicated feature which is enabled by default.

This diff means that we would not support http2 anymore for our REST calls.

I believe it should be reactivated.

Reply:
but do we really need it? `cargo build --all-features` compiles fine without "http2" and "rustls-tls" features.

Reply:
While we don't strictly need it, I think its better to enable. The server side of things - which reqwest connects to - supports it as well. So we may as well benefit from it there.